Resources/Prototypes/Entities/Objects/Specific/Salvage/ore_bag_holding.yml

```diff
@@ -0,0 +1,14 @@
+- type: entity
+  name: ore bag of holding
+  id: OreBagOfHolding
+  parent: OreBag
+  description: A robust bag of holding for salvage billionaires and rich miners alike to carry large amounts of ore. Magnetises any nearby ores when attached to a belt.
+  components:
+  - type: Sprite
+    sprite: Objects/Specific/Mining/ore_bag_holding.rsi
+    state: icon
+  - type: Clothing
+    sprite: Objects/Specific/Mining/ore_bag_holding.rsi
+  - type: Storage
+    grid:
+    - 0,0,19,9
```

Comment:
maybe increase magnet range too since expensive

Reply:
> maybe increase magnet range too since expensive

I thought about it, but I don't know how to implement it, I'm still "green"

Reply:
something like this

```yml
- type: MagnetPickup
  range: 2
```